token-locker/Cargo.toml

```
@@ -0,0 +1,37 @@
+[package]
+name = "token-locker"
+version = "0.1.0"
+authors = ["Pagoda <hello@pagoda.co>"]
+edition = "2018"
+publish = false
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[profile.release]
+codegen-units = 1
+# Tell `rustc` to optimize for small code size.
+opt-level = "z"
+lto = true
+debug = false
+panic = "abort"
+overflow-checks = true
+
+[dependencies]
+near-sdk = { git = "https://github.com/near/near-sdk-rs", rev = "613adcab6322e4f830efb4d5bde2a810ae642158" }
+near-contract-standards = { git = "https://github.com/near/near-sdk-rs", rev = "613adcab6322e4f830efb4d5bde2a810ae642158" }
```

This version is not bound to any release. Let's use the one related to v3.1.0 (the latest stable release)

```suggestion
near-sdk = { git = "https://github.com/near/near-sdk-rs", tag = "3.1.0" }
near-contract-standards = { git = "https://github.com/near/near-sdk-rs", tag = "3.1.0" }
```

---

https://github.com/aurora-is-near/rainbow-token-connector/pull/99/commits/9cd323f709a08bb6ce442d9219e12d4692b04267